_posts/2015-08-17-backing-up-redis.md

```diff
@@ -27,7 +27,7 @@ The first mode to consider is the server process dying, either through fault or
 
 #### Host Failure
 
-In this scenario the host your Redis service is running on dies. Be it through kernel panic, DoS, tripping on a power or network cable, or even a truck running into the data center, the host is offline. Another possibility is the Linux OOM Killer killing Redis because the system is in dire need of memory space and Redis is using it.
+In this scenario the host your Redis service is running on dies. Be it through kernel panic, DoS, tripping on a power or network cable, or even a truck running into the data center, the host is offline. Another possibility is the Linux OOM Killer killing Redis because the system is in dire need of memory space and Redis is using it, [although this can be addressed](http://backdrift.org/oom-killer-how-to-create-oom-exclusions-in-linux) and doesn't necessarily result in data loss / unavailability.
 
```

Comment:
as I am describing failure scenarios here I don't see the addition of more detail on OOM options as useful.